[Fixed] Critical Build Errors

I tried building for x64 Windows, but got 7 Errors that would prevent it. 

The error message was always a variation of this:

`Library\PackageCache\com.av.smart-hierarchy@94113c3d21\Base\HierarchySettingsProvider.cs(8,26): error CS0234: The type or namespace name 'EditorGUI' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)`

After removing Smart Hierarchy the build process did complete as it should.

Screenshot of the Console:
![Screenshot 2020-11-11 182659](https://user-images.githubusercontent.com/13683581/98843622-81936600-244b-11eb-8251-2c6db9e2a772.png)

--- a/Base/HierarchySettingsProvider.cs
+++ b/Base/HierarchySettingsProvider.cs
@@ -1,11 +1,9 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
+﻿#if UNITY_EDITOR
+using System;
 using UnityEditor;
 using UnityEditor.UIElements;
 using UnityEngine;
 using UnityEngine.UIElements;
-using static UnityEditor.EditorGUI;
 
 namespace AV.Hierarchy
 {
@@ -104,3 +102,4 @@ namespace AV.Hierarchy
         }
     }
 }
+#endif

